ibmcloud/README.md

```diff
@@ -199,8 +199,8 @@ If you used the Terraform templates in [common](./terraform/common) and [cluster
 >     $ ibmcloud account users
 >     ```
 >     If command `ibmcloud account users` displays multiple user IDs, choose the user ID whose state is `ACTIVE`.
-> - `cos_service_instance_name` is the COS Service Instance Name, optional, default to value based on your `cluster_name` tfvar if not set, `${cluster_name}-cos-service-instance`.
-> - `cos_bucket_name` is the COS Bucket Name, optional, default to value based on your `cluster_name` tfvar if not set, `${cluster_name}-cos-bucket`.
+> - `cos_service_instance_name` is the COS Service Instance Name, optional, uses value based on your `terraform.tfvars` variable if not set defaults to `cos-service-instance`.
```

Comment:
I think the English here doesn't flow very well to me, I'd consider something  like
> `cos_service_instance_name` is the the optional variable for COS Service Instance Name with a 
 default value of `cos-image-instance` if not set in `terraform.tfvars`
> `cos_bucket_name` is the COS Bucket Name, which must be unique.

?

Reply:
Thanks for the suggestion. I have made this note clearer in the README.